frontend/packages/kubevirt-plugin/integration-tests-cypress/view/add-source.ts

```diff
@@ -0,0 +1,35 @@
+import { ProvisionSource } from '../../integration-tests/tests/utils/constants/enums/provisionSource';
```

Comment:
I prefer copying the file into `integration-tests-cypress` directory, let's don't have dependencies on the old tests, wdyt?

Reply:
ok, I agree to not have deps on old tests. But I also dont like to just copy the file. I created a new directory `integration-tests-utils` where we can put pieces which are meant to be shared between new and old tests - and moved `ProvisionSource` there.